arduino/discovery/discoverymanager/discoverymanager.go

```diff
@@ -65,9 +65,15 @@ func (dm *DiscoveryManager) Clear() {
 			logrus.Infof("Closed and removed discovery %s", d.GetID())
 		}
 	}
+	dm.discoveriesRunning = false
```

Comment:
```suggestion
```

this is wrong, if the `discoverymanager` is started it should be kept started, see my other comments